public/swagger-ui/css/print.css

```diff
@@ -0,0 +1,1367 @@
+/* Original style from softwaremaniacs.org (c) Ivan Sagalaev <Maniac@SoftwareManiacs.Org> */
```

Comment:
How does all of this swagger-ui stuff fit in? Is this just an existing open source project dropped into a directory? (That's what it looks like at a glance). Is it essential to the API implementation, or just a convenient way of testing it? Assuming it is a "vendor" type of dependency, would it make sense, rather than including it in the PR, to add an appropriate .gitignore rule and include some kind of task to automatically install it on demand (using grunt or Composer or Phing, depending on which options are most feasible)? As it is, it seems to be introducing a fair amount of noise to this PR.